src/dialogs/InviteUsers.cpp

```diff
@@ -65,6 +65,10 @@ InviteUsers::InviteUsers(QWidget *parent)
 
         connect(inviteeInput_, &TextField::returnPressed, this, &InviteUsers::addUser);
         connect(confirmBtn_, &QPushButton::clicked, [this]() {
+                if (!inviteeInput_->text().isEmpty()) {
```

Comment:
Can you add an extra trim here? Otherwise it may become confusing, if you just have a space in the input.